webext/raptor/runner.js

```diff
@@ -14,7 +14,8 @@
 // to serve out the pages that we want to prototype with. Also
 // update the manifest content 'matches' accordingly
 
-
+var runningOnFirefox = true;
```

Comment:
I'd call this `var browserName = "firefox"`, this way it'll be easier to extend the webextension for other browsers in the future.